### ISSUE TITLE
Controlling Access Based on Locales broken

### ISSUE THREAD

**Issue by iznotek** [s00d/nuxt-i18n-micro#21]
**Describe the bug**
Hope to constraint a nuxt page to a particular language
from https://s00d.github.io/nuxt-i18n-micro/guide/per-component-translations#use-cases section 1.

**To Reproduce**
```
<script setup>
const { $defineI18nRoute, $t: t } = useNuxtApp()

$defineI18nRoute({
  locales: ['fr'] // Only these locales are allowed for this route
})
</script>
```

**The error**
`Invalid defineI18nRoute configuration format: { locales: [ 'fr' ] } in file:  /Users/pi/Documents/Code/iznow/deep/deeps/packages/i18n/.playground/pages/fr.vue`

**Expected behavior**
I discovered the bug while trying to translate one page in fr and another in en ...??
hopping ... the page in french ... is there a way to translate a page in a language different from the current configured language ??

**Comment by s00d** [comment]
Hello. Fixed in version 1.19.2.

**Comment by iznotek** [comment]
Great !! 
is there a way to translate a page or a component or just a field t() in a language different from the current configured language ??
thanks
